monitoring/src/minting-monitor.ts

```diff
@@ -178,4 +299,238 @@ export class MintingMonitor implements SystemEventMonitor {
 
     return 6
   }
+
+  private async checkDesignatedMintersHealth(cache: ChainDataCache) {
+    const systemEvents: SystemEvent[] = []
+
+    systemEvents.push(...this.checkDesignatedMintersRequests(cache))
+    systemEvents.push(
+      ...(await this.checkDesignatedMintersFinalizations(cache))
+    )
+
+    return systemEvents
+  }
+
+  private checkDesignatedMintersRequests(cache: ChainDataCache): SystemEvent[] {
+    return cache.mintingRequestedEvents
+      .map((mre) => ({
+        ...mre,
+        designatedMinter: this.getDesignatedMinter(
+          cache.minters,
```

Comment:
If there were updates to the minters' list in the meantime we may end up with false positive errors. We should be using the minters' list from the block when the transaction we're verifying happened.

Or we could just ignore it and expect false alarms to be issued around the time the minters' list is updated.